pyproject.toml

```diff
@@ -19,8 +20,8 @@ dependencies = [
 [project.scripts]
 snappy-device-agent = "snappy_device_agents.cmd:main"
 
-[tool.setuptools.packages.find]
-include = ["snappy_device_agents", "devices"]
+[tool.setuptools.package-data]
+snappy_device_agents = ["data/pi-desktop/*"]
```

Comment:
are you sure that the path is correct? Shouldn't it be "src/snappy_device_agents/data/pi-desktop"?